script.module.autocompletion/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.autocompletion" name="AutoCompletion library" version="2.0.6" provider-name="Philipp Temminghoff (phil65), bbaronSVK, finkleandeinhorn, berkhornet">
+<addon id="script.module.autocompletion" name="AutoCompletion library" version="2.1.0" provider-name="Philipp Temminghoff (phil65), bbaronSVK, finkleandeinhorn, berkhornet">
```

Comment:
Do you mind changing the Lang attributes to en_GB instead of en, while you're at it?